NEWS.md

```diff
@@ -242,13 +242,13 @@ details of the default gems or bundled gems.
     * Kernel#caller, Thread::Backtrace::Location’s methods, etc. are also changed accordingly.
 
         Old:
-        ```
+        ```text
```

Comment:
You seem to have mixed up doc improvements and the diff from https://github.com/ruby/ruby/pull/12322.

Reply:
When you're "stacking" PRs, you usually point the stacked PR to the base branch you're stacking on (which doesn't work if you do that from a fork, so you have to give up in this case). What you're doing is filing PRs with duplicated changes to the same branch. Since you're actually not stacking PRs, you should rebase your branch and remove duplication. 

As Stan said on Slack, your changes don't need to be stacked because markdown files don't have dependencies unlike code and these changes work correctly without each other.